server/app.js

```diff
@@ -32,6 +32,15 @@ io.on('connection', (socket) => {
     console.log("Client connected.");          
     socket.emit("connected", "Hello from server");
 
+    socket.on('disconnect', function () {
+        // find the user based on socket id
+        let user = UserUtils.getUserBySocketId(socket.id);
+        // disconnect the user from the system
+        UserUtils.updateUserLoginInfo(user, false, "null");
```

Comment:
What's "null"?

Reply:
Like an empty string but I didn't want to put an empty string since I wanted it to be easy to read through the text file